Gemfile

```diff
@@ -19,8 +19,10 @@ gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 gem 'reek'
+gem 'factory_bot'
 gem 'rubocop'
 
+
```

Comment:
✂️

Reply:
This is also reported by CC -> `Extra empty line detected at block body beginning. `